pennylane/labs/trotter_error/product_formulas/error.py

```diff
@@ -122,6 +123,14 @@ def perturbation_error(
             objects to compute the perturbation error from
         states (Sequence[AbstractState]): the states to compute expectation values from
         delta (float): time step for the trotter error operator.
+        num_workers (int): the number of concurrent units used for the computation. Default value is set to 1.
```

Comment:
Will there be any parallelization if the default value is kept? If not, you might want to reflect it in the docstring.